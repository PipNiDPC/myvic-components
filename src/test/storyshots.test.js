import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  suite: 'YourvicStoryshots',
  configPath: './src/.storybook'
})
