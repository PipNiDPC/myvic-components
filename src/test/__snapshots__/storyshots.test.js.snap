// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YourvicStoryshots Molecules/Map Map with family violence data 1`] = `
<div
  class="yourvic-map"
>
  <h2
    class="yourvic-map__title"
  >
    Title
  </h2>
   
  <div
    class="rpl-markup tide-wysiwyg app-wysiwyg"
  >
    <div
      class="rpl-markup__inner"
    >
      Description
    </div>
  </div>
   
  <p
    class="rpl-visually-hidden"
  >
    The following is an interactive map component, showing the location of programs being run in support of the Prevention of Family Violence initiative.
  </p>
   
  <div>
    <div
      class="yourvic-map__container"
    >
      <div
        class="yourvic-map__map-container"
      >
        <button
          class="rpl-button full-screen-btn rpl-button--primary"
        >
          <span>
            <span
              class="rpl-text-icon__group"
            >
              <!---->
              Enter
            </span>
             
            <span>
               full screen
            </span>
          </span>
        </button>
         
        <div
          class="yourvic-map__container__expand-btn-container"
        >
          <button
            class="rpl-button expand-btn rpl-button--primary"
          >
            <!---->
          </button>
        </div>
         
        <div
          class="yourvic-map-core"
        >
           
          <div
            class="yourvic-map-core__container"
          >
            <div
              class="yourvic-map-core__map"
              id="map"
            />
          </div>
        </div>
      </div>
       
      <div
        class="yourvic-map__sidebar-container"
      >
        <div
          class="yourvic-map-sidebar-home"
        >
          <div
            class="yourvic-map-sidebar-header"
          >
            <div
              class="yourvic-map-sidebar-header__ribbon"
            />
             
            <div
              class="yourvic-map-sidebar-header__content"
            >
              <!---->
               
              <h1>
                0 family violence prevention projects across Victoria
              </h1>
               
              <p>
                
              </p>
               
              <a
                class="yourvic-map-sidebar-header__btn"
              >
                <span
                  class="rpl-text-label rpl-text-label--dark rpl-text-label--dark--underline rpl-text-label--small rpl-text-label--small--underline"
                  iconcolor="primary"
                >
                  View all projects
                </span>
              </a>
            </div>
          </div>
           
          <div
            class="yourvic-map-sidebar-home__content"
          >
            <!---->
             
            <div>
              <div>
                <div
                  class="yourvic-map-categories-page__menu"
                >
                  <span
                    class="rpl-text-label yourvic-map-categories-page__header-btn rpl-text-label--small rpl-text-label--emphasis"
                  >
                    
      Browse by category
    
                  </span>
                   
                  <a
                    class="yourvic-map-categories-page__other-btn"
                  >
                    <span
                      class="rpl-text-label rpl-text-label--small rpl-text-label--small--underline rpl-text-label--emphasis"
                      iconcolor="primary"
                    >
                      
      Browse by area
      
                    </span>
                  </a>
                </div>
                 
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Aboriginal
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Arts
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Bystander
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Children and young people
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Culturally linguistically diverse
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Disability focus
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Education, children and young people
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Health, family and community services
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Higher education
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        LGBQTI
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Media
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Parents
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Public spaces, infrastructure and facilities
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Research or action research
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Seniors
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Sports recreation
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Technology online
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
                <a>
                  <div
                    class="yourvic-map-card"
                  >
                    <div
                      class="yourvic-map-card__header"
                    >
                      <!---->
                       
                      <span
                        class="yourvic-map-card__title"
                      >
                        Workplace cultural change
                      </span>
                       
                      <div
                        class="yourvic-map-card__arrow"
                      >
                        <!---->
                      </div>
                    </div>
                     
                  </div>
                </a>
              </div>
            </div>
             
            <!---->
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`YourvicStoryshots Molecules/Map Map with free wifi data 1`] = `
<div
  class="free-wifi-map demo-content"
>
  <div
    class="yourvic-map-core"
  >
     
    <div
      class="yourvic-map-core__container"
    >
      <div
        class="yourvic-map-core__map"
        id="map"
      />
    </div>
  </div>
</div>
`;

exports[`YourvicStoryshots Molecules/Map Map with no data 1`] = `
<div
  class="yourvic-map-core"
>
   
  <div
    class="yourvic-map-core__container"
  >
    <div
      class="yourvic-map-core__map"
      id="map"
    />
  </div>
</div>
`;
